client/src/test/scala/org/http4s/client/middleware/FollowRedirectSpec.scala

```diff
@@ -16,7 +16,8 @@ class FollowRedirectSpec extends Http4sSpec with Tables {
 
   val service = HttpService[IO] {
     case req @ _ -> Root / "ok" =>
-      Ok(req.body).putHeaders(
+      Ok(
```

Comment:
Alternate proposal, without variadic headers:

```scala
      Ok(
        req.body,
         Headers(
           Header("X-Original-Method", req.method.toString),
           Header("X-Original-Method", req.method.toString),
           Header(
             "X-Original-Content-Length",
             req.headers.get(`Content-Length`).fold(0L)(_.length).toString),
           Header("X-Original-Authorization", req.headers.get(Authorization.name).fold("")(_.value))
           Header("X-Original-Authorization", req.headers.get(Authorization.name).fold("")(_.value))
       )
     )
)
```

Yeah, this one was ugly anyway.